apps/condo/domains/common/components/Highliter.tsx

```diff
@@ -7,14 +7,18 @@ interface IHighliterProps {
     renderPart: (part: string, index: number) => React.ReactElement
 }
 
+const escapeRegex = (text: string) => {
+    return text.replace(/[-[\]{}()*+?.,\\/^$|#\s]/g, '\\$&')
```

Comment:
Can we move this regexp to constants with a description of what it is doing? Maybe, someone will need it later

Reply:
done 👍